ray-operator/test/e2e/support.go

```diff
@@ -175,3 +178,54 @@ func jobSubmitterPodTemplateApplyConfiguration() *corev1ac.PodTemplateSpecApplyC
 						corev1.ResourceMemory: resource.MustParse("500Mi"),
 					}))))
 }
+
+func IsEnvVarExisted(env string, pod *corev1.Pod) bool {
+	for _, envVar := range pod.Spec.Containers[utils.RayContainerIndex].Env {
+		if envVar.Name == env {
+			return true
+		}
+	}
+	return false
+}
+
+func deployRedisWithoutPassword(test Test, g *WithT, namespace string) {
+	deployRedis(test, g, namespace, "")
+}
+
+func deployRedis(test Test, g *WithT, namespace string, password string) {
```

Comment:
nit: make the naming style consistent?
```suggestion
func deployRedis(t Test, g *WithT, namespace string, password string) {
```